wordpress-mobile/WordPress-Aztec-iOS
# Group toolbar elements on the iPad.

This PR, groups the toolbar format elements by category, so you have the following the groups

 - Media
 - Character attributes: strong, em, underline, strikethrough
 - Paragraph attributes: blockquote, list ordered, list unordered
 - Link
 - Code

This is only visible on non .compact screen sizes.

How to test:
 - Run the demo app
 - On the app test the different orientations and multitask modes
 - Test also on the iPhone normal and plus to see if all works ok.
